packages/mobile/android/app/build.gradle

```diff
@@ -203,6 +203,9 @@ android {
     }
     buildTypes {
         debug {
+            // uncomment this to test proguard in debug builds
+            // minifyEnabled true
+            proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro", "devsupport.pro"
```

Comment:
Should this be commented out then?

Reply:
Ah yes good catch. I think if minify is not enabled that line won't do anything, but just to be safe I commented it out. Thanks!